operations/node_management/Rebuild_NCNs/Wipe_Drives.md

```diff
@@ -0,0 +1,83 @@
+# Wipe Disks
+
+**Warning:** This is the point of no return. Once the disks are wiped, the node must be rebuilt.
+
+All commands in this section must be run on the node being rebuilt \(unless otherwise indicated\). These commands can be done from the ConMan console window.
+
+Only follow the steps in the section for the node type that is being rebuilt:
+
+- [Wipe Disks](#wipe-disks)
+  - [Wipe Disks: Master](#wipe-disks-master)
+  - [Wipe Disks: Worker Node](#wipe-disks-worker-node)
+  - [Wipe Disks: Utility Storage Node](#wipe-disks-utility-storage-node)
+
+## Wipe Disks: Master
+
+1. Unmount the etcD volume and remove the volume group.
+
+   **NOTE:** etcd should already be stopped as part of the "Prepare Master Node" steps.
+
+   ```bash
+   /run/lib-etcd
+   vgremove -f etcdvg0-ETCDK8S
+   ```
+
+2. Unmount the `SDU` mountpoint and remove the volume group.
+
+   ```bash
+   umount /var/lib/sdu
+   vgremove -f metalvg0-CRAYSDU
+   ```
+
+3. Wipe the drives
+
+   ```bash
+   mdisks=$(lsblk -l -o SIZE,NAME,TYPE,TRAN | grep -E '(sata|nvme|sas)' | sort -h | awk '   {print "/dev/" $2}')
+   wipefs --all --force $mdisks
+   ```
+
+## Wipe Disks: Worker Node
+
+1. Stop contianerd, unmount volumes, and wipe drives.
```

Comment:
I find it odd when there's only 1 member of a numbered list. I think there's must be something missing.